lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -1171,6 +1172,57 @@ bool HandshakeBuilder::visitHandshake(MuxOp op) {
   return true;
 }
 
+bool HandshakeBuilder::visitHandshake(handshake::SelectOp op) {
+  ValueVector selectSubfields = portList[0];
+  Value selectValid = selectSubfields[0];
+  Value selectReady = selectSubfields[1];
+  Value selectData = selectSubfields[2];
+
+  ValueVector resultSubfields = portList[3];
+  Value resultValid = resultSubfields[0];
+  Value resultReady = resultSubfields[1];
+  Value resultData = resultSubfields[2];
+
+  ValueVector trueSubfields = portList[1];
+  Value trueValid = trueSubfields[0];
+  Value trueReady = trueSubfields[1];
+  Value trueData = trueSubfields[2];
+
+  ValueVector falseSubfields = portList[2];
+  Value falseValid = falseSubfields[0];
+  Value falseReady = falseSubfields[1];
+  Value falseData = falseSubfields[2];
+
```

Comment:
I think we've enabled c++17, in which case you can do
```auto& [falseValid, falseReady, falseData] = portList[2]```
Not required by any means, but might be nicer.

Reply:
Thank you for the tip! I think, for now, i'll keep it as is since this pattern is repeated 10's of times within `HandshakeToFIRRTL.cpp`. Such style change might be better suited for a separate refactor - we nevertheless plan to rewrite `HandshakeToFIRRTL.cpp` to Handshake-to-RTL, wherein modernization is definitely due.